db/migrate/20161014133811_move_content_view_version_description_to_histories.rb

```diff
@@ -1,12 +1,13 @@
 class MoveContentViewVersionDescriptionToHistories < ActiveRecord::Migration
-  class Katello::ContentViewVersion < ApplicationRecord
+  class Katello::FakeContentViewVersion < ApplicationRecord
```

Comment:
As you mentioned mostly to prevent confusion with external module. The fact that you had to rename it to "Fake*" tells me that chances of conflicting with global namespace is possible. Anyway I agree not much risk here. I just think it would look simpler and the name space is unnecessary.  Keep it optional as a change. ACK